Test/Tests/RealtimeClientChannelTests.swift

```diff
@@ -903,7 +903,12 @@ class RealtimeClientChannelTests: XCTestCase {
             }
         }
 
-        expect(client.connection.state).toEventually(equal(ARTRealtimeConnectionState.connected), timeout: testTimeout)
+        waitUntil(timeout: testTimeout) { done in
```

Comment:
You seem to have accidentally included the contents of #1989?

Reply:
Yes, removed.